src/content/migrate/3.md

```diff
@@ -14,14 +14,13 @@ contributors:
   - howdy39
   - selbekk
   - ndelangen
+  - EugeneHlushko
 ---
 
 The following sections describe the major changes from webpack 1 to 2.
 
 T> Note that there were far fewer changes between 2 and 3, so that migration shouldn't be too bad. If you are running into issues, please see [the changelog](https://github.com/webpack/webpack/releases) for details.
 
-W> This content may be moved to the blog post in the near future as version 2 has been out for a while. On top of that, version 3 was recently released and version 4 is on the horizon. As noted above, folks should instead to refer to [the changelog](https://github.com/webpack/webpack/releases) for migrations.
```

Comment:
Take a look at changes done in [#1939](https://github.com/webpack/webpack.js.org/pull/1939/files#diff-91aa65efa4b8d708cd748987e4397532R49).